presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestOracleConfig.java

```diff
@@ -33,6 +33,9 @@ public void testDefaults()
                 .setVarcharMaxSize(4000)
                 .setTimestampDefaultPrecision(6)
                 .setNumberDefaultScale(10)
+                .setConnectionPoolEnabled(false)
```

Comment:
Is there a reason to have connection pooling disable by default? I would suggest having `true` here.

Reply:
none.. changed